lib/dm-do-adapter/adapter.rb

```diff
@@ -475,7 +475,7 @@ def delete_statement(query)
         #
         # @api private
         def columns_statement(properties, qualify)
-          properties.map { |property| property_to_column_name(property, qualify) }.join(', ')
+          properties.map { |property| chainable_property_to_column_name(property, qualify) }.join(', ')
```

Comment:
Lets change this back per our discussion with updating `property_to_column_name`